pkg/model/stack.go

```diff
@@ -777,9 +783,22 @@ func (svcResources *ServiceResources) IsDefaultValue() bool {
 	return svcResources.CPU.Value.IsZero() && svcResources.Memory.Value.IsZero() && svcResources.Storage.Size.Value.IsZero() && svcResources.Storage.Class == ""
 }
 
-func isPathAComposeFile(path string) bool {
+// isFileCompose checks if the path is a compose file
+// if the env var OKTETO_SUPPORT_STACKS_ENABLED is set to true, it will return true for any file no matter the name
+// if the env var is not set, it will return true for files that start with "compose", "docker-compose" or "okteto-compose"
+func isFileCompose(path string) bool {
 	base := filepath.Base(path)
-	return strings.HasPrefix(base, "compose") || strings.HasPrefix(base, "docker-compose") || strings.HasPrefix(base, "okteto-compose")
+	isComposeFileName := strings.HasPrefix(base, "compose") || strings.HasPrefix(base, "docker-compose") || strings.HasPrefix(base, "okteto-compose")
+	isStackSupported := env.LoadBooleanOrDefault(stackSupportEnabledEnvVar, false)
```

Comment:
@jLopezbarb can we keep `true` as default? this way we can announce this change on the next release and give time to our users to adapt to this change. And we could change the default in CLI 3.0

Reply:
Also, could we track if a file is managed as an Okteto Stack or Docker Compose? This will help us to make the decision of removing Okteto Stacks, or defaulting to Docker Compose

Reply:
> can we keep true as default? this way we can announce this change on the next release and give time to our users to adapt to this change. And we could change the default in CLI 3.0

Done in https://github.com/okteto/okteto/pull/4254/commits/11abd9133b11698e27db082700eb6696f9fbbc89

 > Also, could we track if a file is managed as an Okteto Stack or Docker Compose? This will help us to make the decision of removing Okteto Stacks, or defaulting to Docker Compose
 
@pchico83  We are already tracking it in mixpannel for okteto deploys, should we track it as a separate event?

Reply:
No, that fine. Do we need to adapt the logic of the metric to consider the envvar?

Reply:
No, we get if it's a compose or a stack based on this code so there is no need to change it elsewhere